website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+description: Create one or more indexes on a collection to improve query performance and enable certain query features.
+---
+
+# `createIndexes` Command
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateUniqueIndex from '!!raw-loader!../../guides/requests/create-indexes/1-create-unique-index.js';
+import CreateCompoundIndex from '!!raw-loader!../../guides/requests/create-indexes/2-create-compound-index.js';
+import CreatePartialIndex from '!!raw-loader!../../guides/requests/create-indexes/3-create-partial-index.js';
+
+<!-- Category for Command "Administrative, Diagnostic, and Utility Commands" should be organized per folder as present in the compatibility document -->
+
+<!-- Short description of the command and its usage. -->
+
+Creates one or more indexes on the specified collection.
+
+Indexes improve query performance and enable certain query features (e.g., uniqueness, sorting, partial matches).
+
+<!-- Syntax structure of the command -->
+
+<!-- markdownlint-disable MD033 -->
+
+```js
+db.runCommand({
+  createIndexes: <collection>,
+  indexes: [
+    {
+      key: <index specification>,
+      name: <index name>,
+      unique: <boolean>,
+      partialFilterExpression: <filter expression>,
+      expireAfterSeconds: <integer>
+    },
+    // ... more index specifications
+  ]
+})
+```
+
+This command accepts a list of index specifications and applies them to a given collection.
+
+<!-- majorly options such as arguments,modieifers, stage, operators, parameters, options supported by the command in a table (if necessary or available). Takes in the following parameters: -->
+
+| Argument/Parameter        | Type    | Description                                                                  |
+| ------------------------- | ------- | ---------------------------------------------------------------------------- |
+| `createIndexes`           | string  | The name of the collection on which to create the index.                     |
+| `indexes`                 | array   | An array of index specification objects. Each object defines one index.      |
+| `key`                     | object  | The fields to index and their sort order or type (`1`, `-1`, `"text"`, etc). |
```

Comment:
Note how it references "key", not "index specification".
And what is "object"? Our concepts documentation tells nothing about objects.